modules/table/src/main/java/org/apache/ignite/internal/table/TableImpl.java

```diff
@@ -107,12 +109,22 @@ public SchemaRegistry schemaView() {
 
     /** {@inheritDoc} */
     @Override public Collection<Tuple> getAll(Collection<Tuple> keyRecs) {
-        throw new UnsupportedOperationException("Not implemented yet.");
+        return sync(getAllAsync(keyRecs));
     }
 
     /** {@inheritDoc} */
     @Override public @NotNull CompletableFuture<Collection<Tuple>> getAllAsync(Collection<Tuple> keyRecs) {
-        throw new UnsupportedOperationException("Not implemented yet.");
+        Objects.requireNonNull(keyRecs);
```

Comment:
Please add here and below @NotNull annotation and also reflect that in TableView methods and java docs, as we did for KeyValueView.java